src/components/ReactMic.js

```diff
@@ -81,7 +97,6 @@ export default class ReactMic extends Component {
   }
 
   visualize = () => {
-    const self = this
```

Comment:
Unused variable

Reply:
Thanks Ryan, this PR is great.  Just one error that I fixed, but other than that...merged!